install.sh

```diff
@@ -28,7 +44,6 @@ git clone https://github.com/AlessandroYorba/Despacio.git && cd despacio.git/col
 
 cp -f $PWD/gitignore_global ~/.gitignore_global;
```

Comment:
rewrite always? move this line after `. ./update-gitconfig.sh`

Reply:
Could you please open a PR for this?